tests/languages/java-file/app/MainApp.java

```diff
@@ -0,0 +1,19 @@
+public class MainApp
+{
+	public static void main(String[] args)
+	{
+		System.out.println("Hello SGX world from MainApp Java!");
+		FileReadSample frs = new FileReadSample();
+		System.out.println("Trying to read file /app/input.txt");
+		String data = frs.readFile("/app/input.txt");
+
+		System.out.println("Content of the file:");
+		System.out.println(data);
+		
+		if (data.compareTo("This is the content of file\n") == 0) {
+		    System.out.println("TEST_PASSED");
```

Comment:
Let's exit with exit code 1 in case of failure, or just throw an exception which has the same effect. Identifiers should only be a last resort in tests.

Reply:
Done. Added exit()